src/components/Song.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const Song = ({currentSong}) => {
   return (
     <div className="song-container">
-      <img src={currentSong.cover}></img>
+      <img alt={currentSong.name} src={currentSong.cover}></img>
```

Comment:
next time make this and the previous commit into one commit like: 
'fixed errors - img alt attribute + null value errors for useState'